src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/config/DigitalPrisonReportingMiExceptionHandler.kt

```diff
@@ -41,6 +39,18 @@ class DigitalPrisonReportingMiExceptionHandler {
         ),
       )
   }
+  private fun respondWithBadRequest(e: Exception): ResponseEntity<ErrorResponse> {
+    log.info("Validation exception: {}", e.message)
```

Comment:
Minor - would warn be a more appropriate log level here? (Or even error if this was severe enough).

Reply:
I think Error would not be appropriate as this is a user input validation error from which the service recovers. So Warn or Info seem right. This was actually set to info already by @dominic-messenger-moj  and I just refactored the code and moved it to a method.